experimental/java-microprofile-dev-mode/image/Dockerfile-stack

```diff
@@ -0,0 +1,43 @@
+FROM adoptopenjdk/openjdk8-openj9
+
+USER root
+RUN  apt-get -qq update \
+  && apt-get -qq install -y curl maven \
+  && DEBIAN_FRONTEND=noninteractive apt-get -qq upgrade -y \
+  && apt-get -qq clean \
+  && rm -rf /tmp/* /var/lib/apt/lists/*
+
+COPY ./config /config
+COPY ./project /project
+
+WORKDIR /project/
+RUN mkdir -p /mvn/repository
+RUN mvn -B -Dmaven.repo.local=/mvn/repository -N io.takari:maven:wrapper -Dmaven=$(mvn help:evaluate -Dexpression=maven.version -q -DforceStdout)
+RUN mvn -B -Dmaven.repo.local=/mvn/repository install dependency:go-offline -DskipTests
+RUN mvn -Dmaven.repo.local=/mvn/repository install dependency:go-offline -DskipTests -f pom-dev.xml
+
+WORKDIR /project/user-app
+
+ENV APPSODY_MOUNTS="~/.m2/repository:/root/.m2/repository;src:/project/user-app/src;pom.xml:/project/user-app/pom.xml"
+ENV APPSODY_DEPS=/project/user-app/target
+
+ENV APPSODY_WATCH_IGNORE_DIR=/project/user-app/target
+ENV APPSODY_WATCH_REGEX="(pom.xml)$"
+
+ENV APPSODY_INSTALL="../validate.sh && mvn install -DskipTests && /project/install-dev-deps.sh"
+
+ENV APPSODY_RUN="mvn liberty:dev -DskipTests"
```

Comment:
`-DskipTests` should be removed.  If users run Appsody in interactive mode `appsody run interactive`, this will allow them to press Enter to run tests.